llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -7801,6 +7598,15 @@ VectorizationFactor LoopVectorizationPlanner::computeBestVF() {
 
       InstructionCost Cost = cost(*P, VF);
       VectorizationFactor CurrentFactor(VF, Cost, ScalarCost);
+
+      // Make sure that the VF doesn't use more than the number of available
+      // registers
```

Comment:
nit: this comment can be dropped, because the code below says as much.

Reply:
Done.